src/Application/Commands/TakeRegularGivingDonations.php

```diff
@@ -236,9 +234,17 @@ private function confirmPreCreatedDonationsThatHaveReachedPaymentDate(
                     } else {
                         throw $exception;
                     }
+                } catch (MandateCollectionRepeatedlyFailed $exception) {
```

Comment:
I would have preferred to do this in an 'upstream' service but it was fiddly to do correctly without introducing a circular dependency.